### PR TITLE
doc: Add 'Zephyr FSM plan' to glossary

### PR DESCRIPTION
**Pull request by romkell**
Add 'Zephyr FSM plan' to glossary
as used in the zephyr functional safety management plan.

### CODE REVIEW

**Review by henrikbrixandersen — COMMENTED** [review]
"FSM" is a well-established abbreviation of "Finite-state machine". I would be very careful in reusing it for something like this within the Zephyr project.

**Comment by ubieda** [comment]
Agree on not re-using the FSM acronym

File: doc/glossary.rst [hunk 1]
```diff
@@ -180,4 +180,9 @@ Glossary of Terms
       term storage rather than copying it into RAM, saving writable memory for
       dynamic data and not the static program code.
 
+   Zephyr FSM Plan
+      This Safety Plan / Functional Safety Management Plan contains the planning
+      and mapping of methods and resources of the activities of the Zephyr Safety
+      Lifecycle.
```
Comment (simhein) [inline doc/glossary.rst:186]:
```suggestion
   Zephyr safety plan
      The safety plan also known as functional safety management plan contains the planning
      and mapping of methods and resources of the activities of the Zephyr safety
      lifecycle.
```

**Review by simhein — CHANGES_REQUESTED** [review]

**Review by kartben — CHANGES_REQUESTED** [review]
@simhein's suggestion goes in the right direction. This new entry needs to be reference from at least somewhere though, as we don't want to add entries to the glossary just for the sake of it, but rather have these entries being pointed to from other pages in the docs.

**Comment by simhein** [comment]
> @simhein's suggestion goes in the right direction. This new entry needs to be reference from at least somewhere though, as we don't want to add entries to the glossary just for the sake of it, but rather have these entries being pointed to from other pages in the docs.

True I generally agree with that, I will check on it how we can integrate the safety plan and reference to it in the docs because we can't publish the whole plan because of legal issues with references and text to the safety standard

**Comment by kartben** [comment]
closing this PR as it looks like there is agreement on taking a different approach than adding safety terms in the general Zephyr glossary

**Pull request closed without merging**